frontend-web/webclient/app/Applications/Group.tsx

```diff
@@ -20,53 +25,67 @@ const ApplicationsGroup: React.FunctionComponent = () => {
         null
     );
 
+    const dispatch = useDispatch();
+    const [, invokeCommand] = useCloudCommand();
```

Comment:
If you are not using `loading` from `useCloudCommand`, I think this can be replace with `callAPI`